modular_skyrat/modules/gladiator/items.dm

```diff
@@ -0,0 +1,62 @@
+/obj/item/crusher_trophy/gladiator
+	name = "ashen bones"
+	desc = "A set of soot-coated ribs from a worthy warrior. Suitable as a trophy for a kinetic crusher."
+	icon_state = "demon_claws"
+	gender = PLURAL
+	denied_type = /obj/item/crusher_trophy/gladiator
+	bonus_value = 10
+
+/obj/item/crusher_trophy/gladiator/effect_desc()
+	return "the crusher to have a <b>[bonus_value]%</b> chance to block any incoming attack."
+
+/obj/item/crusher_trophy/gladiator/add_to(obj/item/kinetic_crusher/incomingchance, mob/living/user)
+	. = ..()
+	if(.)
+		incomingchance.block_chance += bonus_value
+
+/obj/item/crusher_trophy/gladiator/remove_from(obj/item/kinetic_crusher/incomingchance, mob/living/user)
+	. = ..()
+	if(.)
+		incomingchance.block_chance -= bonus_value
+
+/obj/item/claymore/agateram //it just works
+	name = "ancient blade agateram"
+	desc = "A millenia-old blade made from a material that you can't even begin to fathom. It flows with the power of the Marked One who once held it." //That thing was too big to be called a sword. Too big, too thick, too heavy, and too rough, it was more like a large hunk of iron.
+	icon = 'modular_skyrat/master_files/icons/obj/agateram.dmi'
+	icon_state = "demonsword"
+	inhand_icon_state = "demonsword"
+	lefthand_file = 'modular_skyrat/master_files/icons/mob/agateraminhandsleft.dmi'
+	righthand_file = 'modular_skyrat/master_files/icons/mob/agateraminhandsright.dmi'
+	w_class = WEIGHT_CLASS_HUGE
+	slot_flags = null
+	force = 20
+	resistance_flags = INDESTRUCTIBLE
+	var/faction_bonus_force = 100
+	var/static/list/nemesis_factions = list("mining", "boss")
+	armour_penetration = 20
+	block_chance = 30
+	sharpness = SHARP_EDGED
+
+/obj/item/gun/magic/staff/spellblade/agateram/attack(mob/living/target, mob/living/carbon/human/user)
```

Comment:
You've defined this on ``/obj/item/claymore/agateram`` but you're using it on ``/obj/item/gun/magic/staff/spellblade/agateram``